proto/mixer.proto

```diff
@@ -589,6 +589,20 @@ message InterestingPlaceAspect {
   repeated Aspect aspects = 1;
 }
 
+// Request for GetStat service.
+message GetStatValueRequest {
+  // dcid of the place.
+  string place = 1;
```

Comment:
do we prefer place over dcid? just to note that every api except for GetStatRequest uses dcid or dcids, rather than place. thoughts?